backend/app/tests/services/users/test_utils.py

```diff
@@ -28,3 +37,150 @@ def test_check_creator_github_handle_not_in_list(self, github_data, projects, pu
         username = github_data(username="invalid")["username"]
         assert not any(username in project["creator_github_handle"] for project in projects())
         assert not any(username in publication["creator_github_handle"] for publication in publications())
+
+
+@pytest.fixture
+def mock_user():
+    return mUser(
+        id=1,
+        github_id=12345,
+        username="test_user",
+        email="test@example.com",
+        avatar_url=None,
+        role="reviewer",
+        created_on=datetime.datetime.utcnow(),
+        last_login=datetime.datetime.utcnow(),
+        is_active=True,
+        remarks=[],
+    )
```

Comment:
Move this to `conftest.py`. You could use existing fixtures to populate some fields.